grails-bom/plugins.properties

```diff
@@ -1,12 +1,12 @@
+cache=8.0.0-SNAPSHOT
+fieldsPluginVersion=6.0.0-SNAPSHOT
```

Comment:
I think the property must be `fields=6.0.0-SNAPSHOT` as the artifactId is taken from the property name.

Reply:
Whoops!  You shouldn't have pre-approved lol

Reply:
That's what happens when I copy and paste while talking to 3 people at the same time. 🤦